packages/browser/src/lib/priority-queue/persisted.ts

```diff
@@ -86,16 +86,17 @@ function mutex(key: string, onUnlock: Function, attempt = 0): void {
 }
 
 export class PersistedPriorityQueue extends PriorityQueue<Context> {
-  constructor(maxAttempts: number, key: string) {
+  constructor(maxAttempts: number, key: string, writeKey: string) {
```

Comment:
Nit: I feel like it would be cleaner if this class didn't know what a writekey was (since it's part of our  domain logic, and this is hypothetically a generic). Also, we wouldn't have to repeat writekey:key an anymore!

Reply:
Originally required it to handle the fallback case of reading keys that didn't include the `writeKey`, but I can remove that from this now. It did give me the benefit of having TypeScript yell at me if any usage didn't include the writekey though, but that work is done now.